README.md

```diff
@@ -472,7 +472,7 @@ export class TestService {
     private readonly signer: EthersSigner,
```

Comment:
Alternatively, could rename this variable to `ethersSigner` to be consistent with other examples.
```suggestion
    private readonly ethersSigner: EthersSigner,
```

Reply:
Perfect. Done.